spec/sse/workers/splits_worker_spec.rb

```diff
@@ -309,7 +309,7 @@
       "trafficTypeName": "user",
       "excluded":{
         "keys":["mauro@split.io","gaston@split.io"],
-        "segments":[]
+        "segments":["segment1"]
```

Comment:
Update to `"segments":[{ "type": "standard", "name": "segment1"}]` if not done in a different PR.

Reply:
will do